benchkit/commandwrappers/taskset.py

```diff
@@ -49,5 +55,12 @@ def command_prefix(
                 cpu_order_list=cpu_order_list,
             )
 
-        cmd_prefix = ["taskset", "--cpu-list", str(mtc)] + cmd_prefix
+        if self.set_all_cpus and nb_threads is not None:
+            cpu_order_list = [str(x) for x in cpu_order_list[0:nb_threads]]
```

Comment:
correct me if i'm wrong but in this codepath cpu_order_list can be uninitialized right? If cpu_order is None?

Reply:
You're correct

Reply:
Changed the code to account for that